l10n_it_location_nuts/tests/__init__.py

```diff
@@ -1,3 +0,0 @@
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-
-from . import test_nuts
```

Comment:
come mai lo elimini?

Reply:
Danno errore sulla 14, sono stati eliminati anche qui:  https://github.com/OCA/l10n-spain/tree/14.0/l10n_es_location_nuts

Reply:
Hai mica un riferimento dove viene eliminato e se ne parla?

Reply:
> Danno errore sulla 14, sono stati eliminati anche qui: https://github.com/OCA/l10n-spain/tree/14.0/l10n_es_location_nuts

A quanto vedo nel modulo linkato i test non ci sono mai stati, nemmeno nelle versioni precedenti.
Nel modulo italiano invece furono aggiunti dall'inizio in https://github.com/OCA/l10n-italy/pull/582

Reply:
> Ok, ma danno errore. Come risolviamo?

È normale che durante una migrazione ci sia da correggere qualcosa, in questo caso vanno corretti i test

Reply:
Suggerimento ben accetto ..